### PR TITLE
semver: replaced . with - for versioning in dev

### PR DESCRIPTION
**Pull request by hiteshbedre**

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ phases:
         else
             LATEST_TAG=$(curl -i "Accept: application/vnd.github+json" -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/Privado-Inc/privado/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
             VERSION_INITIAL=${LATEST_TAG:1}
-            VERSION=$VERSION_INITIAL.$CODEBUILD_BUILD_NUMBER
+            VERSION=$VERSION_INITIAL-$CODEBUILD_BUILD_NUMBER
```
Comment (ojaswa1942) [inline buildspec.yml:22]: Build numbers are supposed to be prefixed with `+`.
`-` indicates prerelease label.

Ref: https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions

**Review by ojaswa1942 — CHANGES_REQUESTED** [review]

File: buildspec.yml (hunk 1 above)
Comment (pandurangpatil) [reply buildspec.yml:22]: @ojaswa1942 we also tag the docker image with the given version and it doesn't allow to have `+` in tag name. So for now I am going ahead and approving this change, we may have to revisit this again.

**Review by pandurangpatil — APPROVED** [review]
As docker tags don't support the `+` in the tag name. I am approving this change to proceed further.

**Comment by pandurangpatil** [comment]
@ojaswa1942 you need to approve this change to proceed further.

**Review by ojaswa1942 — APPROVED** [review]

**Pull request merged**